backend/alerts/integrations/discord/discord.go

```diff
@@ -24,7 +24,7 @@ func NewDiscordBot(guildId string) (*Bot, error) {
 	session, err := discordgo.New("Bot " + DiscordBotSecret)
 
 	if err != nil {
-		return nil, errors.Wrap(err, "error creating Discord session")
+		return nil, errors.Join(err, errors.New("error creating Discord session"))
```

Comment:
Just an example to prove that the Go upgrade worked (`errors.Join` is only available in 1.20).

In a follow up PR, this would simply be:

```go
if err != nil {
  return nil, err
}
```

Reply:
ah, neat!